packages/story-editor/src/components/secondaryPopup/index.js

```diff
@@ -79,6 +85,8 @@ function Popup({
           role="dialog"
           aria-label={ariaLabel}
           state={state}
+          anchor={anchor}
```

Comment:
In the design-system `Popup`, anchor is a `ref` of the element that has the popup and `spacing` in the format of `{ x, y }` is used for spacing and `placement` is used to define if it's on the left (or somewhere else) relative to the anchor. Could we use the same logic here for consistency? E.g. could we use `spacing` with the relevant values and/or `placement` instead of `anchor`. Thoughts?

Reply:
We should be good here now. @merapi 's idea to add anchor to control the left vs right value was great, that combined with an override in the layer set up and we're gucci. No need to further tweak anything.

Reply:
I'll update this to `placement`